pageserver/src/consumption_metrics.rs

```diff
@@ -287,6 +287,12 @@ pub async fn collect_metrics_iteration(
                     }
                 } else {
                     error!("metrics endpoint refused the sent metrics: {:?}", res);
+                    for metric in chunk_to_send.iter() {
+                        // Report if the metric value is suspiciously large
+                        if metric.value > 2 ^ 40 {
```

Comment:
`2 ^ 40 == 42` 

is it really large? have you meant `pow`?

Reply:
yeah, it was a thinko
Fixed